File: OOP Lab Final/lab7.cpp
#include <bits/stdc++.h>
using namespace std;
class students
{
public:
   int roll;
   string name;
   string section;
   void details()
   {
      cout << "Enter name of the students :";
      cin >> name;
      cout << "Enter section of the students :";
      cin >> section;
      cout << "Enter roll of the students :";
      cin >> roll;
   }
   void get_deatils()
   {
      cout << "Information of the students" << endl;
      cout << "Name = " << name << endl;
      cout << "Section = " << section << endl;
      cout << "Roll = " << roll << endl;
   }
};
int main()
{
   students student;
   student.details();
   student.get_deatils();

   return 0;
}

File: OOP_C++_JAVA/lab7.java
import java.util.Scanner;

class Students {
    public int roll;
    public String name;
    public String section;

    public void details() {
        Scanner sc = new Scanner(System.in);
        System.out.print("Enter name of the student: ");
        name = sc.nextLine();
        System.out.print("Enter section of the student: ");
        section = sc.nextLine();
        System.out.print("Enter roll of the student: ");
        roll = sc.nextInt();
        sc.close();
    }

    public void getDetails() {
        System.out.println("Information of the student");
        System.out.println("Name = " + name);
        System.out.println("Section = " + section);
        System.out.println("Roll = " + roll);
    }
}
public class lab7 {
    public static void main(String[] args) {
        Students student = new Students();
        student.details();
        student.getDetails();
    }
    
}
